### PR TITLE
Use VADS in Documentation

### PR DESCRIPTION
**Pull request by mjlumetta**
https://github.com/department-of-veterans-affairs/vets-contrib/issues/3011

### CODE REVIEW

File: src/containers/documentation/CategoryPage.tsx [hunk 1]
```diff
@@ -52,8 +53,8 @@ export default class CategoryPage extends React.Component<RouteComponentProps<IA
     }
 
     return (
-      <section role="region" aria-labelledby={headerId} className="api-overview">
-        <h1 id={headerId}>{categoryName}</h1>
+      <section role="region" aria-labelledby={headerId} className="va-api-api-overview">
```
Comment (jasalisbury) [inline src/containers/documentation/CategoryPage.tsx]: looks like `va-api-api-overview` can be removed.

File: src/containers/documentation/DocumentationRoot.tsx [hunk 2]
```diff
@@ -93,7 +93,7 @@ function ExploreSideNav() {
 export default class DocumentationRoot extends React.Component<RouteComponentProps<IApiNameParam>, {}> {
   public render() {
     return (
-      <div className={classNames('explore', 'vads-u-padding-y--5')}>
+      <div className={classNames('documentation', 'vads-u-padding-y--5')}>
```
Comment (jasalisbury) [inline src/containers/documentation/DocumentationRoot.tsx]: Can't seem to find this class anywhere

File: src/containers/documentation/CategoryPage.tsx (hunk 1 above)
Comment (mjlumetta) [reply src/containers/documentation/CategoryPage.tsx]: re: this and your other comment - there are like ~2 styles that use `va-api-api-overview` and `documentation`. i forgot to commit the new `Documentation.scss`

File: src/containers/releaseNotes/ReleaseNotes.tsx [hunk 3]
```diff
@@ -12,8 +12,6 @@ import { IApiNameParam } from '../../types';
 import CategoryReleaseNotesPage from './CategoryReleaseNotesPage';
 import ReleaseNotesOverview from './ReleaseNotesOverview';
 
-import '../Documentation.scss';
```
Comment (kalilsn) [inline src/containers/releaseNotes/ReleaseNotes.tsx]: 🙌

**Pull request merged**